src/Management/src/Endpoint/ManagementWebApplicationBuilderExtensions.cs

```diff
@@ -32,6 +33,7 @@ public static class ManagementWebApplicationBuilderExtensions
     /// </param>
     public static WebApplicationBuilder AddDbMigrationsActuator(this WebApplicationBuilder applicationBuilder)
     {
+        applicationBuilder.UseCloudHosting(ManagementWebHostBuilderExtensions.ConfigureManagementUrls);
```

Comment:
As you've previously commented, the methods in this file are a bit brute-force/repetetive... would it make sense to add a private method that calls `UseCloudHosting`, `ActivateActuatorEndpoints` (and any other future common calls) and returns the applicationBuilder as a short-term solution to reduce some of the bulk?